### PR TITLE
fix(OpenGLMapper): Add some test to avoid error when several mapper a…

### PR DESCRIPTION
**Pull request by laurennlam**
Add some test to avoid error when several mapper are instanciated

### CODE REVIEW

File: Sources/Rendering/OpenGL/PolyDataMapper/index.js [hunk 1]
```diff
@@ -78,11 +78,11 @@ export function vtkOpenGLPolyDataMapper(publicAPI, model) {
     // user specified pre replacements
     const openGLSpec = model.renderable.getViewSpecificProperties().OpenGL;
     let shaderReplacements = null;
-    if (openGLSpec !== undefined) {
+    if (openGLSpec !== undefined && openGLSpec !== null) {
```
Comment (jourdain) [inline Sources/Rendering/OpenGL/PolyDataMapper/index.js]: Why not do `if (openGLSpec)` instead?

File: Sources/Rendering/OpenGL/PolyDataMapper/index.js [hunk 2]
```diff
@@ -78,11 +78,11 @@ export function vtkOpenGLPolyDataMapper(publicAPI, model) {
...
       shaderReplacements = openGLSpec.ShaderReplacements;
     }
 
-    if (shaderReplacements !== null) {
+    if (shaderReplacements !== undefined && shaderReplacements !== null) {
```
Comment (jourdain) [inline Sources/Rendering/OpenGL/PolyDataMapper/index.js]: Why not do `if (shaderReplacements)` instead?

File: Sources/Rendering/OpenGL/PolyDataMapper/index.js [hunk 3]
```diff
@@ -97,7 +97,7 @@ export function vtkOpenGLPolyDataMapper(publicAPI, model) {
     publicAPI.replaceShaderValues(shaders, ren, actor);
 
     // user specified post replacements
-    if (shaderReplacements !== null) {
+    if (shaderReplacements !== undefined && shaderReplacements !== null) {
```
Comment (jourdain) [inline Sources/Rendering/OpenGL/PolyDataMapper/index.js]: Why not do `if (shaderReplacements)` instead?

**Comment by laurennlam** [comment]
I'll change it. I didn't know that `if(blabla)` checked both 'undefined' and 'null'

**Comment by aronhelser** [comment]
Well, it checks for any 'falsey' value, which includes null and undefined, but also includes 0 and '' (empty string). As long as you are sure your variable contains either an object or undefined/null, then if (blabla) is a good check. If it's a string or number, you have to watch out for '' and 0.

**Comment by laurennlam** [comment]
Thanks for the explanations @aronhelser !

**Pull request merged**